src/node.ts

```diff
@@ -36,6 +36,17 @@ class Node extends VEventTarget {
    */
   resolvedInputs = 0;
 
+  /**
+   * The state of the node.
+   */
+  state: { [key: string]: unknown; } | null = {};
+
+  /**
+   * A boolean used to determine if the state gets reset before the next cycle
+   * @private
```

Comment:
`@private` doesn't serve any purpose - use that on a variable, `private resetState = true`

Reply:
_From TypeScript 3.8 onwards, you can use JSDoc to modify the properties in a class. First are the accessibility modifiers: @public, @private, and @protected. These tags work exactly like public, private, and protected respectively work in TypeScript._ https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html

Reply:
After a test, I can confirm that it works.

Reply:
Changed to private instead @private to reduce confusion.